util/helpers.py

```diff
@@ -7,3 +7,27 @@ def secs_to_hours(secs):
     hours, remainder = divmod(secs, 3600)
     minutes, seconds = divmod(remainder, 60)
     return '%d:%02d:%02d' % (hours, minutes, seconds)
+
+def check_ctcdecoder_version():
+    import sys # pylint: disable=import-outside-toplevel
+    import os # pylint: disable=import-outside-toplevel
+    import semver # pylint: disable=import-outside-toplevel
+
+    ds_version_s = open(os.path.join(os.path.dirname(__file__), '../VERSION')).read().strip()
+
+    try:
+        from ds_ctcdecoder import __version__ as decoder_version # pylint: disable=import-outside-toplevel
+    except ImportError as e:
+        if e.msg.find('__version__') > 0:
+            print("DeepSpeech version ({}) requires CTC decoder to expose __version__. Please install proper ds_ctcdecoder package".format(ds_version_s))
```

Comment:
s/install proper ds_ctcdecoder package/upgrade the ds_ctcdecoder package to version {}/